ado_wrapper/resources/runs.py

```diff
@@ -16,7 +16,9 @@
 
 class RunAllDictionary(TypedDict):
     template_variables: dict[str, Any]
```

Comment:
Obviously this was previously wrongly named, perhaps rename template_variables to just `variables`?